src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.FeatureCollection.cs

```diff
@@ -264,15 +264,15 @@ bool IHttpUpgradeFeature.IsUpgradableRequest
 
         int IFeatureCollection.Revision => _featureRevision;
 
-        IPAddress IHttpConnectionFeature.RemoteIpAddress { get; set; }
+        IPAddress RemoteIpAddress { get; set; }
 
-        IPAddress IHttpConnectionFeature.LocalIpAddress { get; set; }
+        IPAddress LocalIpAddress { get; set; }
 
-        int IHttpConnectionFeature.RemotePort { get; set; }
+        int RemotePort { get; set; }
 
-        int IHttpConnectionFeature.LocalPort { get; set; }
+        int LocalPort { get; set; }
 
-        string IHttpConnectionFeature.ConnectionId { get; set; }
+        new string ConnectionId { get; set; }
```

Comment:
Yikes. I would duplicate the property and have the new properties in Frame.cs pass through to here like we do for `IHttp(Request|Response)Features`.

This class only contains explicit interface implementations.